server/views/temporary-accommodation/assessments/change-date.njk

```diff
@@ -0,0 +1,51 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "../../partials/showErrorSummary.njk" import showErrorSummary %}
+{% from "../components/ta-date-input/macro.njk" import taDateInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = content.docTitle + " - " + applicationName %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block beforeContent %}
+    {{ govukBackLink({
+        text: "Back",
+        href: paths.assessments.full({ id: assessment.id })
+    }) }}
+{% endblock %}
+
+{% block content %}
+    {% set hintHtml %}
+        {% if content.hint %}
+            <p class="govuk-hint">{{ content.hint }}</p>
+        {% endif %}
+        <p class="govuk-hint">{{ dateInputHint('future') }}</p>
+    {% endset %}
+
+    <h1 class="govuk-heading-l">{{ content.title }}</h1>
+
+    {{ showErrorSummary(errorSummary) }}
```

Comment:
If I understand the code correctly, shouldn't the error summary be above the h1?

https://design-system.service.gov.uk/components/error-summary/#where-to-put-the-error-summary

Reply:
I believe you are correct, and for some reason this pattern was not adopted for this service (a quick search shows all error summaries show _after_ the h1) -- probably one to fix as a separate ticket!